configure.ac

```diff
@@ -110,7 +111,16 @@ case "$host" in
         LT_EXTRA="-os2dllname SDL2img"
         ;;
 esac
-AM_CONDITIONAL(USE_IMAGEIO, test x$enable_imageio = xyes)
+case "$host" in
+    *-*-mingw*)
+        AC_SUBST([MINGW],1)
+        ;;
+    *)
+        AC_SUBST([MINGW],0)
+        ;;
+esac
+AM_CONDITIONAL(USE_IMAGEIO, $enable_imageio)
+AC_SUBST([USE_IMAGEIO], test x$enable_imageio = xyes)
```

Comment:
This probably needs to be an actual variable that evaluates to 1 or 0. If you want you can comment this out for your commit and I can fix it up here.